fdbserver/RESTKmsConnector.actor.cpp

```diff
@@ -166,7 +168,7 @@ struct RESTKmsConnectorCtx : public ReferenceCounted<RESTKmsConnectorCtx> {
 
 std::string getFullRequestUrl(Reference<RESTKmsConnectorCtx> ctx, const std::string& url, const std::string& suffix) {
 	if (suffix.empty()) {
-		TraceEvent("GetFullUrlEmptyEndpoint", ctx->uid).log();
+		TraceEvent(SevWarn, "RESTGetFullUrlEmptyEndpoint", ctx->uid).log();
```

Comment:
`SevWarnAlways`?

Reply:
Given URI is user input, I don't think we want to log Warn messages for wrong input, the operation would fail and sufficient logs will be recorded.